docs/deploymentnew/on-prem-installation-guidelines.md

```diff
@@ -180,8 +183,8 @@ A Wireguard bastion host (Wireguard server) provides secure private channel to a
 
 > Note:
 
-* Increase the no. of peers above in case needed more than 30 wireguard client confs (`-e PEERS=30`).
-* Change the directory to be mounted to wireguard docker in case needed. All your wireguard confs will be generated in the mounted directory (`-v /home/ubuntu/wireguard/config:/config`).
+>  *  Increase the no. of peers above in case needed more than 30 wireguard client confs (`-e PEERS=30`).  
```

Comment:
Increase the no. of peers above in case more than 30 wireguard client confs (`-e PEERS=30`) are needed.